### PR TITLE
Splash add impact size data

### PR DESCRIPTION
**Pull request by amrocha**
### WHY are these changes introduced?

Based on feedback we received on the Splash action we're trying to make it more actionable and less noisy

### WHAT is this pull request doing?

1. Stop posting a loading comment on the PR at first. This is pure noise. Now the action posts a comment once it has something relevant to show the user. (the loading comment still shows up on follow-up comments, to not mislead the user into thinking old data is still relevant)
1. Change the text of the comment to be briefer. I removed the feedback line as well since I never received any feedback that way. **Should I add back the line about the comment updating itself?**
1. Add average impact data to the comment. The first thing the comment states is whether a change's splash zone is below average, average, or above average.

The impact data is calculated using the 1st and 3rd quartiles. Anything above the 3rd quartile is higher than average, anything below the 1st quartile is below average.

In order to keep this data up to date I added a script that anyone can run locally that fetches new PRs since the last time the script was run, and calculates updated quartile data.

I considered using quantile regression to build a more accurate model, but decided against it in the end since using simple quartiles was a fairly decent approximation.

### CODE REVIEW

**Comment by amrocha** [comment]
💦 Potential splash zone of changes introduced to `src/**/*.tsx` in this pull request:

<table><tbody>
<tr><th align="left">Files modified</th><td>6</td></tr>
<tr><th align="left">Files potentially affected</th><td>69</td></tr>
</tbody></table>

### Details

<details>
<summary><strong>All files potentially affected (total: 69)</strong></summary>

- [`/src/components/Checkbox/Checkbox.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Checkbox/Checkbox.tsx)
- [`/src/components/ChoiceList/ChoiceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ChoiceList/ChoiceList.tsx)
- [`/src/components/ResourceItem/ResourceItem.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceItem/ResourceItem.tsx)
- [`/src/components/ResourceList/ResourceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/ResourceList.tsx)
- [`/src/components/ResourceList/components/CheckableButton/CheckableButton.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/CheckableButton/CheckableButton.tsx)
- [`/src/components/ResourceList/components/BulkActions/BulkActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/BulkActions/BulkActions.tsx)
- [`/src/components/RadioButton/RadioButton.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RadioButton/RadioButton.tsx)
- [`/src/components/TextField/TextField.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TextField/TextField.tsx)
- [`/src/components/Autocomplete/components/TextField/TextField.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/components/TextField/TextField.tsx)
- [`/src/components/Autocomplete/Autocomplete.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/Autocomplete.tsx)
- [`/src/components/Filters/Filters.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Filters/Filters.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/FilterValueSelector/FilterValueSelector.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/FilterValueSelector/FilterValueSelector.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/FilterCreator/FilterCreator.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/FilterCreator/FilterCreator.tsx)
- [`/src/components/ResourceList/components/FilterControl/FilterControl.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/FilterControl.tsx)
- [`/src/components/ResourceList/components/FilterControl/types.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/types.ts)
- [`/src/utilities/theme/index.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/utilities/theme/index.ts)
- [`/src/components/Portal/Portal.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Portal/Portal.tsx)
- [`/src/components/Popover/Popover.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Popover/Popover.tsx)
- [`/src/components/Button/Button.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Button/Button.tsx)
- [`/src/components/Button/utils.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Button/utils.tsx)
- [`/src/components/AccountConnection/AccountConnection.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/AccountConnection/AccountConnection.tsx)
- [`/src/components/Card/Card.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Card/Card.tsx)
- [`/src/components/CalloutCard/CalloutCard.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/CalloutCard/CalloutCard.tsx)
- [`/src/components/SettingToggle/SettingToggle.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/SettingToggle/SettingToggle.tsx)
- [`/src/components/Card/components/Header/Header.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Card/components/Header/Header.tsx)
- [`/src/components/Card/components/Section/Section.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Card/components/Section/Section.tsx)
- [`/src/components/ActionMenu/components/RollupActions/RollupActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ActionMenu/components/RollupActions/RollupActions.tsx)
- [`/src/components/ActionMenu/ActionMenu.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ActionMenu/ActionMenu.tsx)
- [`/src/components/Page/components/Header/Header.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Page/components/Header/Header.tsx)
- [`/src/components/Page/Page.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Page/Page.tsx)
- [`/src/components/Labelled/Labelled.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Labelled/Labelled.tsx)
- [`/src/components/DropZone/DropZone.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DropZone/DropZone.tsx)
- [`/src/components/RangeSlider/types.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/types.ts)
- [`/src/components/RangeSlider/components/DualThumb/DualThumb.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/components/DualThumb/DualThumb.tsx)
- [`/src/components/RangeSlider/RangeSlider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/RangeSlider.tsx)
- [`/src/components/RangeSlider/components/SingleThumb/SingleThumb.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/components/SingleThumb/SingleThumb.tsx)
- [`/src/components/Select/Select.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Select/Select.tsx)
- [`/src/components/Banner/Banner.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Banner/Banner.tsx)
- [`/src/components/DataTable/components/Navigation/Navigation.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DataTable/components/Navigation/Navigation.tsx)
- [`/src/components/DataTable/DataTable.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DataTable/DataTable.tsx)
- [`/src/components/DatePicker/DatePicker.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DatePicker/DatePicker.tsx)
- [`/src/components/EmptyState/EmptyState.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/EmptyState/EmptyState.tsx)
- [`/src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx)
- [`/src/components/Modal/components/Footer/Footer.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Modal/components/Footer/Footer.tsx)
- [`/src/components/Modal/Modal.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Modal/Modal.tsx)
- [`/src/components/Frame/components/ContextualSaveBar/components/DiscardConfirmationModal/DiscardConfirmationModal.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/ContextualSaveBar/components/DiscardConfirmationModal/DiscardConfirmationModal.tsx)
- [`/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx)
- [`/src/components/Frame/Frame.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/Frame.tsx)
- [`/src/components/Toast/Toast.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Toast/Toast.tsx)
- [`/src/components/Frame/components/Toast/Toast.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/Toast/Toast.tsx)
- [`/src/components/Frame/components/ToastManager/ToastManager.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/ToastManager/ToastManager.tsx)
- [`/src/components/Navigation/components/Message/Message.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Navigation/components/Message/Message.tsx)
- [`/src/components/Navigation/Navigation.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Navigation/Navigation.tsx)
- [`/src/components/PageActions/PageActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/PageActions/PageActions.tsx)
- [`/src/components/ResourceList/components/BulkActions/components/BulkActionButton/BulkActionButton.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/BulkActions/components/BulkActionButton/BulkActionButton.tsx)
- [`/src/components/TopBar/components/Menu/components/Message/Message.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/components/Menu/components/Message/Message.tsx)
- [`/src/components/TopBar/components/Menu/Menu.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/components/Menu/Menu.tsx)
- [`/src/components/TopBar/components/UserMenu/UserMenu.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/components/UserMenu/UserMenu.tsx)
- [`/src/components/TopBar/TopBar.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/TopBar.tsx)
- [`/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx)
- [`/src/components/Autocomplete/components/ComboBox/ComboBox.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/components/ComboBox/ComboBox.tsx)
- [`/src/components/Tabs/Tabs.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Tabs/Tabs.tsx)
- [`/src/components/Sheet/Sheet.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Sheet/Sheet.tsx)
- [`/src/components/Tooltip/Tooltip.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Tooltip/Tooltip.tsx)
- [`/src/components/Pagination/Pagination.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Pagination/Pagination.tsx)
- [`/src/components/AppProvider/AppProvider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/AppProvider/AppProvider.tsx)
- [`/src/components/ThemeProvider/ThemeProvider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ThemeProvider/ThemeProvider.tsx)
- [`/src/components/PolarisTestProvider/PolarisTestProvider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/PolarisTestProvider/PolarisTestProvider.tsx)
</details>

<details>
<summary>📄 <code><strong>documentation/Color system.md (total: 0)</strong></code></summary>

#### Files potentially affected (total: 0)


</details>

<details>
<summary>🎨 <code><strong>src/components/Checkbox/Checkbox.scss (total: 6)</strong></code></summary>

#### Files potentially affected (total: 6)

- [`/src/components/Checkbox/Checkbox.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Checkbox/Checkbox.tsx)
- [`/src/components/ChoiceList/ChoiceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ChoiceList/ChoiceList.tsx)
- [`/src/components/ResourceItem/ResourceItem.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceItem/ResourceItem.tsx)
- [`/src/components/ResourceList/ResourceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/ResourceList.tsx)
- [`/src/components/ResourceList/components/CheckableButton/CheckableButton.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/CheckableButton/CheckableButton.tsx)
- [`/src/components/ResourceList/components/BulkActions/BulkActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/BulkActions/BulkActions.tsx)
</details>

<details>
<summary>🎨 <code><strong>src/components/RadioButton/RadioButton.scss (total: 2)</strong></code></summary>

#### Files potentially affected (total: 2)

- [`/src/components/RadioButton/RadioButton.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RadioButton/RadioButton.tsx)
- [`/src/components/ChoiceList/ChoiceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ChoiceList/ChoiceList.tsx)
</details>

<details>
<summary>🎨 <code><strong>src/components/TextField/TextField.scss (total: 10)</strong></code></summary>

#### Files potentially affected (total: 10)

- [`/src/components/TextField/TextField.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TextField/TextField.tsx)
- [`/src/components/Autocomplete/components/TextField/TextField.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/components/TextField/TextField.tsx)
- [`/src/components/Autocomplete/Autocomplete.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/Autocomplete.tsx)
- [`/src/components/Filters/Filters.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Filters/Filters.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/FilterValueSelector/FilterValueSelector.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/FilterValueSelector/FilterValueSelector.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/FilterCreator/FilterCreator.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/FilterCreator/FilterCreator.tsx)
- [`/src/components/ResourceList/components/FilterControl/FilterControl.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/FilterControl.tsx)
- [`/src/components/ResourceList/ResourceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/ResourceList.tsx)
- [`/src/components/ResourceList/components/FilterControl/types.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/types.ts)
</details>

<details>
<summary>🎨 <code><strong>src/styles/foundation/_focus-ring.scss (total: 0)</strong></code></summary>

#### Files potentially affected (total: 0)


</details>

<details>
<summary>🧩 <code><strong>src/utilities/theme/tokens.ts (total: 65)</strong></code></summary>

#### Files potentially affected (total: 65)

- [`/src/utilities/theme/index.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/utilities/theme/index.ts)
- [`/src/components/Portal/Portal.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Portal/Portal.tsx)
- [`/src/components/Popover/Popover.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Popover/Popover.tsx)
- [`/src/components/Button/Button.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Button/Button.tsx)
- [`/src/components/Button/utils.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Button/utils.tsx)
- [`/src/components/AccountConnection/AccountConnection.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/AccountConnection/AccountConnection.tsx)
- [`/src/components/Card/Card.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Card/Card.tsx)
- [`/src/components/CalloutCard/CalloutCard.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/CalloutCard/CalloutCard.tsx)
- [`/src/components/SettingToggle/SettingToggle.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/SettingToggle/SettingToggle.tsx)
- [`/src/components/Card/components/Header/Header.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Card/components/Header/Header.tsx)
- [`/src/components/Card/components/Section/Section.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Card/components/Section/Section.tsx)
- [`/src/components/ActionMenu/components/RollupActions/RollupActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ActionMenu/components/RollupActions/RollupActions.tsx)
- [`/src/components/ActionMenu/ActionMenu.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ActionMenu/ActionMenu.tsx)
- [`/src/components/Page/components/Header/Header.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Page/components/Header/Header.tsx)
- [`/src/components/Page/Page.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Page/Page.tsx)
- [`/src/components/Labelled/Labelled.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Labelled/Labelled.tsx)
- [`/src/components/TextField/TextField.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TextField/TextField.tsx)
- [`/src/components/Autocomplete/components/TextField/TextField.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/components/TextField/TextField.tsx)
- [`/src/components/Autocomplete/Autocomplete.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/Autocomplete.tsx)
- [`/src/components/Filters/Filters.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Filters/Filters.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/FilterValueSelector/FilterValueSelector.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/FilterValueSelector/FilterValueSelector.tsx)
- [`/src/components/ResourceList/components/FilterControl/components/FilterCreator/FilterCreator.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/components/FilterCreator/FilterCreator.tsx)
- [`/src/components/ResourceList/components/FilterControl/FilterControl.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/FilterControl.tsx)
- [`/src/components/ResourceList/ResourceList.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/ResourceList.tsx)
- [`/src/components/ResourceList/components/FilterControl/types.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/FilterControl/types.ts)
- [`/src/components/DropZone/DropZone.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DropZone/DropZone.tsx)
- [`/src/components/RangeSlider/types.ts`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/types.ts)
- [`/src/components/RangeSlider/components/DualThumb/DualThumb.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/components/DualThumb/DualThumb.tsx)
- [`/src/components/RangeSlider/RangeSlider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/RangeSlider.tsx)
- [`/src/components/RangeSlider/components/SingleThumb/SingleThumb.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/RangeSlider/components/SingleThumb/SingleThumb.tsx)
- [`/src/components/Select/Select.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Select/Select.tsx)
- [`/src/components/Banner/Banner.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Banner/Banner.tsx)
- [`/src/components/DataTable/components/Navigation/Navigation.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DataTable/components/Navigation/Navigation.tsx)
- [`/src/components/DataTable/DataTable.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DataTable/DataTable.tsx)
- [`/src/components/DatePicker/DatePicker.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/DatePicker/DatePicker.tsx)
- [`/src/components/EmptyState/EmptyState.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/EmptyState/EmptyState.tsx)
- [`/src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx)
- [`/src/components/Modal/components/Footer/Footer.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Modal/components/Footer/Footer.tsx)
- [`/src/components/Modal/Modal.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Modal/Modal.tsx)
- [`/src/components/Frame/components/ContextualSaveBar/components/DiscardConfirmationModal/DiscardConfirmationModal.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/ContextualSaveBar/components/DiscardConfirmationModal/DiscardConfirmationModal.tsx)
- [`/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx)
- [`/src/components/Frame/Frame.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/Frame.tsx)
- [`/src/components/Toast/Toast.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Toast/Toast.tsx)
- [`/src/components/Frame/components/Toast/Toast.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/Toast/Toast.tsx)
- [`/src/components/Frame/components/ToastManager/ToastManager.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Frame/components/ToastManager/ToastManager.tsx)
- [`/src/components/Navigation/components/Message/Message.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Navigation/components/Message/Message.tsx)
- [`/src/components/Navigation/Navigation.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Navigation/Navigation.tsx)
- [`/src/components/PageActions/PageActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/PageActions/PageActions.tsx)
- [`/src/components/ResourceItem/ResourceItem.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceItem/ResourceItem.tsx)
- [`/src/components/ResourceList/components/BulkActions/BulkActions.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/BulkActions/BulkActions.tsx)
- [`/src/components/ResourceList/components/BulkActions/components/BulkActionButton/BulkActionButton.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ResourceList/components/BulkActions/components/BulkActionButton/BulkActionButton.tsx)
- [`/src/components/TopBar/components/Menu/components/Message/Message.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/components/Menu/components/Message/Message.tsx)
- [`/src/components/TopBar/components/Menu/Menu.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/components/Menu/Menu.tsx)
- [`/src/components/TopBar/components/UserMenu/UserMenu.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/components/UserMenu/UserMenu.tsx)
- [`/src/components/TopBar/TopBar.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/TopBar/TopBar.tsx)
- [`/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx)
- [`/src/components/Autocomplete/components/ComboBox/ComboBox.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Autocomplete/components/ComboBox/ComboBox.tsx)
- [`/src/components/Tabs/Tabs.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Tabs/Tabs.tsx)
- [`/src/components/Sheet/Sheet.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Sheet/Sheet.tsx)
- [`/src/components/Tooltip/Tooltip.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Tooltip/Tooltip.tsx)
- [`/src/components/Pagination/Pagination.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/Pagination/Pagination.tsx)
- [`/src/components/AppProvider/AppProvider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/AppProvider/AppProvider.tsx)
- [`/src/components/ThemeProvider/ThemeProvider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/ThemeProvider/ThemeProvider.tsx)
- [`/src/components/PolarisTestProvider/PolarisTestProvider.tsx`](https://github.com/Shopify/polaris-react/blob/focus_ring-animation-improvements/src/components/PolarisTestProvider/PolarisTestProvider.tsx)
</details>

---

This comment automatically updates as changes are made to this pull request.
Feedback, troubleshooting: open an issue or reach out on Slack in [#polaris-tooling](https://shopify.slack.com/messages/CCNUS0FML).

**Comment by amrocha** [comment]
💦 Potential splash zone of changes introduced to \`${CODEBASE_GLOB}\` in this pull request:

❌ Something went wrong with the Discoverability Github Action. This doesn't stop you from shipping your changes.

<details>
<summary>Details:</summary>

<code>
${text}
</code>
</details>

We’ve been notified and will take a look.
If you need immediate help, ping @amrocha or @kaelig on Slack

**Comment by amrocha** [comment]
⏳ Please wait while I’m computing the dependency graph…
This comment will automatically update in a few minutes

**Comment by amrocha** [comment]
@chloerice @alex-page Can you guys take a look at this PR? This adds average impact data to the splash action. The comments on the PR are all the possible states for the comment. I reworked the content with @sadiesaurus , let me know if you have any feedback on that as well

**Comment by amrocha** [comment]
Note: I modified unrelated files to get the comment to post on this PR, I'll remove all those commits before merging

**Comment by sadiesaurus** [comment]
🤓A couple of fresh content opinions/nitpicks:
- Move away from 3 sentences that start with "This"
- Change the tense of "modify" so it sounds more current and less final  
- Change "could" to "might" (less ambiguous for some ESL speakers)

Rewrite could be: `This pull request modifies 11 files and might impact 140 other files. Because this is a larger than average splash zone for a change, remember to tophat areas that could be affected.`

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export interface ButtonProps {
    * @default 'medium'
    */
   size?: Size;
-  /** Changes the inner text alignment of the button */
+  /** Changes the inner text alignment ofssthe button */
```
Comment (alex-page) [inline src/components/Button/Button.tsx]:
```suggestion
  /** Changes the inner text alignment of the button */
```

**Comment by amrocha** [comment]
Updated the content as per your comment @sadiesaurus 

I'm gonna merge this and follow up with a PR adding storybook links

**Pull request merged**